documentation/src/main/asciidoc/topics/proc_creating_caches_hotrod_onaccess.adoc

```diff
@@ -50,5 +50,5 @@ infinispan.client.hotrod.cache.my-other-cache.configuration_uri=file:/path/to/co
 
 .Reference
 
-* link:{javadocroot}/org/infinispan/client/hotrod/configuration/package-summary.html[`RemoteCacheManager` Javadoc]
-* link:{javadocroot}/org/infinispan/client/hotrod/configuration/RemoteCacheConfigurationBuilder.html[`RemoteCacheConfigurationBuilder` Javadoc]
+* link:{javadocroot}/org/infinispan/client/hotrod/configuration/package-summary.html[Hot Rod Client Configuration]
```

Comment:
since you changed the example, I would add a link to `DefaultTemplate` javadoc. So, the user knows which templates are available.